### PR TITLE
Add float version of HLSL mod

### PR DESCRIPTION
**Pull request by SabinT**
Completing the set :)

### CODE REVIEW

File: math/mod.hlsl [hunk 1]
```diff
@@ -11,6 +12,10 @@ license: |
 
 #ifndef FNC_MOD
 #define FNC_MOD
+float2 mod(float x, float y) {
```
Comment (patriciogonzalezvivo) [inline math/mod.hlsl:15]: I think there is a typo here. This one should return `float` and not `float2`. Right? 
Thank you so much for contributing!

Comment (SabinT) [reply math/mod.hlsl:15]: Oh whoops, thanks 😅 will fix.

Comment (SabinT) [reply math/mod.hlsl]: Done

**Comment by patriciogonzalezvivo** [comment]
Thank you! This is great!

**Pull request merged**